### DISCUSSION TITLE
Separate Functions and procedures from the single folder.

### DISCUSSION THREAD

**Discussion by hardikhari96** [dbeaver/dbeaver#14807] [Q&A]
Separate Functions and procedures from the single folder.

**Comment by LonwoLonwo** [comment]
Hello @hardikhari96 

What DBeaver version do you use?
What MariaDB version do you use?

Did you try to find your functions in the Procedures tab?

![2021-11-19 11_20_26-DBeaver Ultimate 21 2 5 - AccessType_202009031659](https://user-images.githubusercontent.com/45152336/142589531-dddf7e4d-fa01-43e6-bc03-c0574bd6f97d.png)

**Reply by hardikhari96** [reply]
Yes, I found it, can we separate it for a better user experience?
